docs/ref-info/glossary-uid.md

```diff
@@ -176,6 +180,23 @@ import Link from '@docusaurus/Link';
 <dt><MdxJumpAnchor id="gl-closed-operator"><a href="#gl-closed-operator">Closed Operator</a></MdxJumpAnchor></dt>
 <dd>Closed Operator is another term for a <a href="#gl-private-operator">Private Operator</a>.</dd>
 
+<dt><MdxJumpAnchor id="gl-confidential-computing"><a href="#gl-confidential-computing">Confidential Computing (GCP)</a></MdxJumpAnchor></dt>
+<dd>A Confidential Computing solution from Google Cloud Platform (GCP), Confidential Space, is supported for hosting a <a href="#gl-private-operator">Private Operator</a>.</dd>
```

Comment:
Should this be: 
A Confidential Computing solution from Google Cloud Platform (GCP), Confidential Space, **that** is supported for hosting a ...
without the bold, just added to show what I am adding

Reply:
You're right. Thx. Fixed.